### ISSUE TITLE
Null Object Reference crash just sitting there watching battle on 16X

### ISSUE THREAD

**Issue by IBwWG** [larsiusprime/tdrpg-bugs#905]
0.8.4, win7x64, HD mode@1704x960windowed

Hadn't clicked anything for several seconds:

![dq-crash3](https://cloud.githubusercontent.com/assets/16719964/15806021/2148d7b2-2b3a-11e6-847f-628495b8ca89.png)

```
Called from ApplicationMain::main ApplicationMain.hx line 159
Called from ApplicationMain::create ApplicationMain.hx line 71
Called from lime.app.Application::exec lime/app/Application.hx line 222
Called from lime._backend.native.NativeApplication::exec lime/_backend/native/Na
tiveApplication.hx line 136
Called from lime._backend.native.NativeApplication::handleRenderEvent lime/_back
end/native/NativeApplication.hx line 368
Called from lime.app.Event_Void_Void::dispatch lime/app/Event.hx line 145
Called from *::_Function_2_1 lime/app/Application.hx line 110
Called from lime.app.Application::render lime/app/Application.hx line 762
Called from openfl.display.Stage::render openfl/display/Stage.hx line 761
Called from openfl.display.DisplayObjectContainer::__broadcast openfl/display/Di
splayObjectContainer.hx line 355
Called from openfl.display.DisplayObject::__broadcast openfl/display/DisplayObje
ct.hx line 251
Called from openfl.events.EventDispatcher::__dispatchEvent openfl/events/EventDi
spatcher.hx line 237
Called from flixel.FlxGame::onEnterFrame flixel/FlxGame.hx line 530
Called from flixel.FlxGame::step flixel/FlxGame.hx line 663
Called from com.leveluplabs.tdrpg.Main::update com/leveluplabs/tdrpg/Main.hx lin
e 1495
Called from flixel.FlxGame::update flixel/FlxGame.hx line 752
Called from flixel.FlxState::tryUpdate flixel/FlxState.hx line 168
Called from com.leveluplabs.tdrpg.State_Battle::update com/leveluplabs/tdrpg/Sta
te_Battle.hx line 903
Called from com.leveluplabs.tdrpg.State_Battle::updateBasic com/leveluplabs/tdrp
g/State_Battle.hx line 951
Called from com.leveluplabs.tdrpg.BattleField::update com/leveluplabs/tdrpg/Batt
leField.hx line 884
Called from flixel.group.FlxTypedGroup::update flixel/group/FlxGroup.hx line 138

Called from flixel.group.FlxTypedGroup::update flixel/group/FlxGroup.hx line 138

Called from com.leveluplabs.tdrpg.CreatureSprite::update com/leveluplabs/tdrpg/C
reatureSprite.hx line 1625
Called from com.leveluplabs.tdrpg.EnemySprite::updateEffects com/leveluplabs/tdr
pg/EnemySprite.hx line 2160
Called from com.leveluplabs.tdrpg.SwimControl::updateEffects com/leveluplabs/tdr
pg/SwimControl.hx line 57
```

**Comment by IBwWG** [comment]
for battle layout: [export-crash-doing-nothing.zip](https://github.com/larsiusprime/tdrpg-bugs/files/299369/export-crash-doing-nothing.zip)

**Comment by IBwWG** [comment]
Although, quite unfortunately, it seems not to have saved the battle layout for some reason.  I'm pretty sure I was playing on extreme both times...

**Comment by IBwWG** [comment]
[export-crash-doing-nothing2.zip](https://github.com/larsiusprime/tdrpg-bugs/files/299371/export-crash-doing-nothing2.zip)
There, hopefully that has it, I did the same layout again, then quit as soon as the first wave arrived...

**Comment by IBwWG** [comment]
Yep, it does this time.

**Comment by IBwWG** [comment]
Aha, so does the crash.  Voila, a reproducible bug.

**Comment by IBwWG** [comment]
Also on NG bonus 1 (from a NG+ save, playing on NG):
![dq-crash4](https://cloud.githubusercontent.com/assets/16719964/15807982/92ed7b26-2b6b-11e6-8ec3-37c82e7fec30.png)

```
Called from ApplicationMain::main ApplicationMain.hx line 159
Called from ApplicationMain::create ApplicationMain.hx line 71
Called from lime.app.Application::exec lime/app/Application.hx line 222
Called from lime._backend.native.NativeApplication::exec lime/_backend/native/Na
tiveApplication.hx line 136
Called from lime._backend.native.NativeApplication::handleRenderEvent lime/_back
end/native/NativeApplication.hx line 368
Called from lime.app.Event_Void_Void::dispatch lime/app/Event.hx line 145
Called from *::_Function_2_1 lime/app/Application.hx line 110
Called from lime.app.Application::render lime/app/Application.hx line 762
Called from openfl.display.Stage::render openfl/display/Stage.hx line 761
Called from openfl.display.DisplayObjectContainer::__broadcast openfl/display/Di
splayObjectContainer.hx line 355
Called from openfl.display.DisplayObject::__broadcast openfl/display/DisplayObje
ct.hx line 251
Called from openfl.events.EventDispatcher::__dispatchEvent openfl/events/EventDi
spatcher.hx line 237
Called from flixel.FlxGame::onEnterFrame flixel/FlxGame.hx line 530
Called from flixel.FlxGame::step flixel/FlxGame.hx line 663
Called from com.leveluplabs.tdrpg.Main::update com/leveluplabs/tdrpg/Main.hx lin
e 1495
Called from flixel.FlxGame::update flixel/FlxGame.hx line 752
Called from flixel.FlxState::tryUpdate flixel/FlxState.hx line 168
Called from com.leveluplabs.tdrpg.State_Battle::update com/leveluplabs/tdrpg/Sta
te_Battle.hx line 903
Called from com.leveluplabs.tdrpg.State_Battle::updateBasic com/leveluplabs/tdrp
g/State_Battle.hx line 951
Called from com.leveluplabs.tdrpg.BattleField::update com/leveluplabs/tdrpg/Batt
leField.hx line 884
Called from flixel.group.FlxTypedGroup::update flixel/group/FlxGroup.hx line 138

Called from flixel.group.FlxTypedGroup::update flixel/group/FlxGroup.hx line 138

Called from com.leveluplabs.tdrpg.CreatureSprite::update com/leveluplabs/tdrpg/C
reatureSprite.hx line 1625
Called from com.leveluplabs.tdrpg.EnemySprite::updateEffects com/leveluplabs/tdr
pg/EnemySprite.hx line 2160
Called from com.leveluplabs.tdrpg.SwimControl::updateEffects com/leveluplabs/tdr
pg/SwimControl.hx line 57```

**Comment by larsiusprime** [comment]
I reproduced this myself the other day, it should be fixed in 0.8.8
